UICatalog/Scenarios/SpinnerStyles.cs

```diff
@@ -28,7 +27,7 @@ public override void Main ()
 
         foreach (Type style in typeof (SpinnerStyle).GetNestedTypes ())
         {
-            styleDict.Add (i, new KeyValuePair<string, Type> (style.Name, style));
+            styleDict.Add (i, new (style.Name, style));
```

Comment:
Not all the glyphs are rendering well. See the SoccerHeader please.

Reply:
Please open another issue. This exists in v2_develop.